src/code/z_play.c

```diff
@@ -110,34 +110,254 @@
+    frontRoom = transitionActor->sides[0].room;
+    if (frontRoom == transitionActor->sides[1].room) {
+        return false;
+    }
+
+    if (frontRoom == actor->room) {
+        *yaw = actor->shape.rot.y;
+    } else {
+        *yaw = actor->shape.rot.y + 0x8000;
+    }
+
+    return true;
+}
+
+// Unused
+/**
+ * @brief Tests if \p pos is underwater.
+ *
+ * @param[in] globalCtx GlobalContext
+ * @param[in] pos position to test
+ * @return True if inside a waterbox and not above a void.
+ */
+s32 Play_IsUnderwater(GlobalContext* globalCtx, Vec3f* pos) {
+    WaterBox* waterBox;
+    CollisionPoly* poly;
+    Vec3f waterSurfacePos;
+    s32 bgId;
+
+    waterSurfacePos = *pos;
+
+    // == true required to match.
+    if ((WaterBox_GetSurface1(globalCtx, &globalCtx->colCtx, waterSurfacePos.x, waterSurfacePos.z, &waterSurfacePos.y,
+                              &waterBox) == true) &&
+        (pos->y < waterSurfacePos.y) &&
+        (BgCheck_EntityRaycastFloor3(&globalCtx->colCtx, &poly, &bgId, &waterSurfacePos) != BGCHECK_Y_MIN)) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+// z_demo and EnTest4
+// This data appears to be a boolean. It is only set by Play_Init.
+s32 func_8016A168(void) {
+    return D_801D0D50;
+}
+
+// List of cutscene numbers.
+extern s16 D_801D0D64[];
+// s16 D_801D0D64[] = { 0xFFFD, 0xFFFE, 0xFFFC, 0xFFFB, 0xFFF9, 0xFFF5, 0xFFF8, 0xFFF7, 0xFFFA, 0xFFF0 };
+// s16 D_801D0D64[] = { -3, -2, -4, -5, -7, -11, -8, -9, -6, -16 };
+
+// Used by Player
+/**
+ * Sets the cutscene numbers in globalCtx->unk_1879C.
+ * Set to -1 by default. If there is an ActorCutscene where unk4 matches the appropriate element of D_801D0D64 (and
+ * possibly change its priority for the zeroth one)
+ */
+void func_8016A178(GameState* gameState, s32 cutscene) {
+    GlobalContext* globalCtx = (GlobalContext*)gameState;
+    s32 i;
+    s16* phi_s3 = globalCtx->unk_1879C;
+    s16* phi_s1 = D_801D0D64;
+
+    for (i = 0; i < ARRAY_COUNT(globalCtx->unk_1879C); i++, phi_s3++, phi_s1++) {
+        ActorCutscene* actorCutscene;
+        s32 currCutscene;
+
+        *phi_s3 = -1;
+
+        for (currCutscene = cutscene; currCutscene != -1; currCutscene = actorCutscene->additionalCutscene) {
+            actorCutscene = ActorCutscene_GetCutscene(currCutscene);
+
+            if (actorCutscene->unk4 == *phi_s1) {
+                if ((actorCutscene->unk4 == -3) && (actorCutscene->priority == 0x2BC)) {
+                    actorCutscene->priority = 0x226;
+                }
+                *phi_s3 = currCutscene;
+                break;
+            }
+        }
+    }
+}
+
+// These regs are used by Gameplay_Draw, and several actors, purpose as yet unclear.
+void func_8016A268(GameState* gameState, s16 arg1, u8 arg2, u8 arg3, u8 arg4, u8 arg5) {
+    MREG(64) = arg1;
+    MREG(65) = arg2;
+    MREG(66) = arg3;
+    MREG(67) = arg4;
+    MREG(68) = arg5;
+}
```

Comment:
I randomly changed these in game and saw no difference from where I was at, so I'll be curious to find out what these do eventually